hack/local-up-cluster.sh

```diff
@@ -571,6 +574,11 @@ function start_controller_manager {
       node_cidr_args="--allocate-node-cidrs=true --cluster-cidr=10.1.0.0/16 "
     fi
 
+    controllers="*"
+    if [[ "${ENABLE_CONTROLLER_BOOTSTRAP}" = true ]]; then
```

Comment:
I'd rather see an option to let you specify the full controllers param, so we don't have to update this piecemill for new controllers